WebApplicationsTests/UserSearchesTest.cs

```diff
@@ -0,0 +1,222 @@
+﻿
+using Model.Accounts;
+using Model.Hospital;
+using Model.MedicalExam;
+using Model.Schedule;
+using Moq;
+using System;
+using System.Collections.Generic;
+using WebApplication.Backend.DTO;
+using WebApplication.Backend.Repositorys;
+using WebApplication.Backend.Services;
+using Xunit;
+
+namespace WebApplicationsTests
+{
+    public class UserSearchesTest
```

Comment:
Tests are ok as far as logic and structure go, but this is a very nice example of why it's good to extract test objects and instantiate them elsewhere 😄

Reply:
https://stackoverflow.com/questions/29786674/how-to-use-setup-method-to-arrange-for-a-unit-test-how-about-parameters
you can use [SetUp] to make shared arrange section